firefox_puppeteer/docs/index.rst

```diff
@@ -36,8 +36,9 @@ future. Each library is available from an instance of the FirefoxTestCase class.
    ui/about_window/window
    ui/menu
    ui/pageinfo/window
-   ui/tabbar
-   ui/toolbars
+   ui/browser/tabbar
+   ui/browser/toolbars
+   ui/browser/windows
```

Comment:
Oh, sorry! Looks like somehow an additional 's' made it into the suggestion of my last review comments. This should have been `window.py` similar to the page info window as seen right above. Sorry for that.